machine_learning_hep/data/data_run3/database_ml_parameters_LcToPKPi_newformat_mult_ana.yml

```
@@ -51,33 +51,34 @@ LcpKpi:
     read:
       evtorig:
         level: all
-        index: fIndexHf3PCollBases
+        index: fIndexHFLCCOLLBASES
         trees:
-          O2hf3pcollbase: [fNumContrib, fMultZeqNTracksPV, fCentFT0A, fCentFT0C, fCentFT0M, fCentFV0A]
+          O2hflccollbase: [fNumContrib, fMultZeqNTracksPV, fCentFT0M]
         extra:
           fIsEventReject: 0
 
       evtoriggen:
         level: gen
-        index: fIndexHf3PMcCollBases
+        index: fIndexHFLCMCCOLLBASES
         trees:
-          O2hf3pmccollbase: [fPosX, fPosY, fPosZ]
-          O2hf3pmcrcollid: [fIndexArrayHf3PCollBases]
-          
+          O2hflcmccollbase: [fPosX, fPosY, fPosZ, fCentFT0M]
+          O2hflcmcrcollid: [fIndexArrayHFLCCOLLBASES]
+        rename: {old: fCentFT0M, new: fCentFT0Mmc}
```

This syntax does not scale. If this is needed (and I am not sure it is), the rename entry should have a list of entries to rename.
```suggestion
        rename: 
          - fCentFT0M: fCentFT0Mmc
          - fCentTPC: fCentTPCNew
```
If the problem is with the merging, columns existing in both tables get a suffix. Is this not sufficient?

---

Yes, the problem is exactly this. I can change it